#164588447 Update README.md file

lumie31 opened the pull request:
#### What does this PR do?
- Update README.md file
#### Description of Task to be completed?
- I made modifications to the readme file of the project which makes it clear for anyone who checks out the project know what is going on easily
#### How should this be manually tested?
- Visit the repository on GitHub and check the readme section
#### Any background context you want to provide?
- Nil
#### What are the relevant pivotal tracker stories?
- [#164588447](https://www.pivotaltracker.com/n/projects/2315110/stories/164588447)
#### Screenshots (if appropriate)
Nil
#### Questions:
Nil

lumie31 committed:
chore(setup) Update README.md file

- This commit modifies and updates the readme of the project which makes it clear for anyone who checks out the project know what is going on easily

[maintains #164588447]

faksam commented on README.md:
```diff
@@ -4,3 +4,53 @@
 A web application that helps people exchange messages/information over the internet
+
+## Getting Started
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
```
```These instructions will get your copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.```

lumie31 replied:
Alright, will do.

lumie31 replied:
Hello Samuel! I already updated the getting started section of my project's readme as suggested

faksam reviewed (changes requested):
Update the introductory paragraph of your readme.

lumie31 commented:
> Update the introductory paragraph of your readme.

On checking this out, I figured that what currently exists in my `Installing` section of the readme is pretty similar to the idea of the `Getting Started` section, I'm thinking of just taking away the `Getting started` section and leaving the `Installing` section. 
Do you think this is a good idea?

lumie31 committed:
Update Readme Getting started section

- Implement LFA feedback on changing content of the getting started section of readme

[maintains  #164588447]
